### PR TITLE
Cryptojacking Fixes

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Changes the name of the layout to Cryptojacking
Fixes a typo in the If-Then-Else transformer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml [hunk 1]
```diff
@@ -192,9 +192,9 @@ tasks:
           XSOAR has detected activity related to Cryptojacking.
           After careful analysis, XSOAR found the activity malicious.
 
-          Please enter XSOAR incident ID ${incident.id} for further investigation and response.
+          Please enter XSOAR incident ID ${alert.id} for further investigation and response.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:195]: need to change this whole sentence **XSOAR incident ID**

File: Packs/CortexXDR/.pack-ignore [hunk 2]
```diff
@@ -66,4 +66,7 @@ ignore=PB121
 ignore=BA101
 
 [file:playbook-Cortex_XDR_-_Cloud_Cryptomining.yml]
+ignore=BA101
+
+[file:layoutscontainer-Cortex_XDR_-_XCLOUD_Cryptomining.json]
```
Comment (altmannyarden) [inline Packs/CortexXDR/.pack-ignore:71]: What do we have in the layout?

File: Packs/CortexXDR/ReleaseNotes/4_10_17.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+##### Cortex XDR - XCLOUD Cryptojacking
+
+- Changes the name to 'XCLOUD Cryptojacking'
+
+#### Playbooks
+
+##### Cortex XDR - Cloud Cryptojacking
+
+- Fixes a typo in the If-Then-Else transformer
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_10_17.md:16]: I saw this fix only on the XSIAM playbook - not in the XSOAR playbook

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml (hunk 1 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:195]: fixed

File: Packs/CortexXDR/.pack-ignore (hunk 2 above)
Comment (melamedbn) [reply Packs/CortexXDR/.pack-ignore:74]: ignore=BA101

File: Packs/CortexXDR/ReleaseNotes/4_10_17.md (hunk 3 above)
Comment (melamedbn) [reply Packs/CortexXDR/ReleaseNotes/4_10_17.md]: looks good in XSOAR

File: Packs/CortexXDR/.pack-ignore (hunk 2 above)
Comment (altmannyarden) [reply Packs/CortexXDR/.pack-ignore:74]: What does the error mean? :)

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**